ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx

```diff
@@ -196,7 +263,7 @@ export function transformPolicyData(nodes: Node[], flows: number): CustomModel {
     };
     nodes.forEach(({ entity, policyIds, outEdges, nonIsolatedEgress, nonIsolatedIngress }) => {
         const networkPolicyState = getNetworkPolicyState(nonIsolatedEgress, nonIsolatedIngress);
-        const node = getNodeModel(entity, policyIds, networkPolicyState);
+        const node = getNodeModel(entity, policyIds, networkPolicyState, false, outEdges);
```

Comment:
nit: Bare values, like `false` in this line, are hard to understand later. If we always pass the same constant, best to name it.
```
const THIS_VALUE_IS_FALSE_BECAUSE_OF_THIS_REASON = false;
```
```suggestion
        const node = getNodeModel(entity, policyIds, networkPolicyState, THIS_VALUE_IS_FALSE_BECAUSE_OF_THIS_REASON, outEdges);
```